sources/packages/backend/apps/api/src/route-controllers/confirmation-of-enrollment/_tests_/e2e/confirmation-of-enrollment.institutions.confirmEnrollment.e2e-spec.ts

```diff
@@ -87,6 +91,58 @@ describe("ConfirmationOfEnrollmentInstitutionsController(e2e)-confirmEnrollment"
     expect(updatedApplication.applicationStatus).toBe(
       ApplicationStatus.Completed,
     );
+
+    // Assert that the sequence control record was created correctly.
+    const sequenceControl = await sequenceControlRepo.findOne({
+      where: {
+        sequenceName: "Full Time_DISBURSEMENT_DOCUMENT_NUMBER",
+        sequenceNumber: firstDisbursementSchedule.documentNumber.toString(),
+      },
+    });
+    expect(sequenceControl).toBeDefined();
+  });
+
+  it("Should allow the COE confirmation when the application is on Enrolment status and all the conditions are fulfilled for Part Time offering.", async () => {
```

Comment:
Really minor but appointed in other recent PRs, we should try to follow the naming conventions for part-time/full-time as defined in https://github.com/bcgov/SIMS/wiki/Acronyms-&-Terminology#word--spelling-usage
Old code should remain, but new code can follow it.
Please rename `Part Time` to `part-time`/